README.md

```diff
@@ -23,6 +23,19 @@ Pomgen uses the lockfiles generated by [rules_jvm_external](https://github.com/b
 - pomgen < 2.0.0 is compatible with rules_jvm_external < 5.0
 - pomgen >= 2.0.0 is compatible with rules_jvm_external >= 5.0
 
+### Overidding Dependencies
```

Comment:
Can you see if this override file is documented in rules_jvm_external?  This section should just say something like "pomgen supports rule_jvm_external's overrides file ..." and then link to the relevant doc in rules_jvm_external.  Good to have an example here, though, keep that!

Reply:
I have given the link to the rules_jvm_external override deps.